app/helpers/address_helper.rb

```diff
@@ -105,8 +105,9 @@ def sanitize_address(address)
         .with_indifferent_access
         .except(:uprn, :county, :town_city)
         .values
-        .map(&:to_s)
-        .map(&:upcase).join(" ")
+        .select { |value| value.instance_of?(String) }
```

Comment:
This is a good change that'll make the code more robust 👍 

I wonder why `value` here can be something other than a `String`? 🤔

Reply:
At the moment - all of the form inputs are `type=text`.  So, not only will they be converted by the browser into string params, but we do not want anything other than a string in them.  So, I feel removing anything but a string is reasonable at the moment.

Reply:
Of course, this may change in the future!